compiler/rustc_passes/src/loops.rs

```diff
@@ -24,22 +25,38 @@ enum Context {
     Closure(Span),
     AsyncClosure(Span),
     UnlabeledBlock(Span),
+    IfUnlabeledBlock(Span),
```

Comment:
```suggestion
    UnlabeledIfBlock(Span),
```

nit: I think this is a slightly better name.

Reply:
Done. Thank you.